pkgs/development/interpreters/python/build-python-package-setuptools.nix

```diff
@@ -26,13 +26,13 @@ in attrs // {
   buildPhase = attrs.buildPhase or ''
     runHook preBuild
     cp ${setuppy} nix_run_setup
-    ${python.interpreter} nix_run_setup ${lib.optionalString (setupPyBuildFlags != []) ("build_ext " + (lib.concatStringsSep " " setupPyBuildFlags))} bdist_wheel
+    ${python.crossPython.interpreter} nix_run_setup ${lib.optionalString (setupPyBuildFlags != []) ("build_ext " + (lib.concatStringsSep " " setupPyBuildFlags))} bdist_wheel
```

Comment:
cc @Ericson2314 might have an opinion on this.

Reply:
Not sure how to consider the sliding window here. We need to have a reference to the `buildPackages` version, otherwise it gets very messy.

Reply:
Yes good catch and good fix. `pythonForBuild` is a perfect name such that I know the intent is correct :).